tools/ncf.py

```diff
@@ -548,6 +548,8 @@ def generate_technique_content(technique, methods):
   content.append('{')
 
   
+  content.append('  vars:')
+  content.append('    "resource_dir" string => "${this.promise_dir}/resources";')
```

Comment:
I'd use resources in the name variable too for consistency

Reply:
Done :)